Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py

```diff
@@ -9,7 +9,10 @@
         SelectEvents = cms.vstring('pathALCARECOHcalCalMinBias')
     ),
     outputCommands = cms.untracked.vstring(
-	'keep *_gtDigisAlCaMB_*_*',
```

Comment:
the collection gtDigisAlCaMB of digis which you remove from this PR, is actually L1 trigger digis.
_hltTriggerSummaryAOD_ and _TriggerResults_ might give you the same information you're giving up.

Why do you need to add the full RAW information from all the sub detectors, to make up for the missing L1T digis ?